docs/tutorials/12_qaoa_runtime.ipynb

```diff
@@ -5,10 +5,24 @@
    "id": "d9a5a73c",
    "metadata": {},
    "source": [
-    "# QAOA Runtime\n",
-    "\n",
-    "The Qiskit runtime is an execution model that permits us to run an entire program on the backend side. Here, we discuss how to run the QAOA algorithm in the Qiskit runtime. We will discuss several of the features that this first version of the QAOA Runtime makes available.\n",
-    "\n"
+    "# QAOA Runtime"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d492e9c3",
+   "metadata": {},
+   "source": [
+    ".. warning::\n",
+    "  The `VQEClient` and `QAOAClient` discussed in this tutorial is **DEPRECATED** as of version 0.6 of Qiskit Optimization and will be removed no sooner than 3 months after the release. Instead you should use the new primitives based `SamplingVQE` with the Qiskit IBM Runtime Sampler primitive. For more details on how to migrate check out these guides [here (VQE)](https://qisk.it/algo_migration#vqe) and [here (QAOA)](https://qisk.it/algo_migration#qaoa)."
```

Comment:
```suggestion
    "  The `VQEClient` and `QAOAClient` discussed in this tutorial are **DEPRECATED** as of version 0.6 of Qiskit Optimization and will be removed no sooner than 3 months after the release. Instead you should use the new primitives based `SamplingVQE` and `QAOA` algorithms respectively with the Qiskit IBM Runtime Sampler primitive. For more details on how to migrate check out these guides [here (SamplingVQE)](https://qisk.it/algo_migration#vqe) and [here (QAOA)](https://qisk.it/algo_migration#qaoa)." 
```

I added QAOA, also I changed the last part from listing VQE link to SamplingVQE but left the anchor #vqe since that section does talk about SamplingVQE (since VQE no longer works with optimzation I felt it better this way)

Reply:
Thanks!